operations/evaluate.go

```diff
@@ -21,7 +21,7 @@ func Evaluate() cli.Command {
 
 	return cli.Command{
 		Name:  "evaluate",
-		Usage: "reads a project configuration and expands tags and matrix definitions, printing the expanded definitions",
+		Usage: "reads a project configuration and expands tags and included files (excluding module includes), printing the expanded definitions",
```

Comment:
the verbiage is a bit hard for me to parse, can we change `(excluding module includes)` to something more explicit like `(excluding files included from a separate module)`?

Reply:
sure, i couldn't quite decide what here would be right so thanks for the suggestion!